receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper_test.go

```diff
@@ -306,6 +306,7 @@ func (p *processHandleMock) IOCounters() (*process.IOCountersStat, error) {
 	return args.Get(0).(*process.IOCountersStat), args.Error(1)
 }
 
+
```

Comment:
Please make sure no unrelated/accidental changes are done in the PR

Reply:
done